packages/brite-core/src/__tests__/command.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { ILogger } from "..";
+import { BriteCommand } from "../commands";
+import { BriteCommandRunner } from "../runner";
+
+class FakeCommand extends BriteCommand {
+    public defaultEnvironment = 'myenv';
+
+    public async execute() {
+        return {
+            code: 0,
+        };
+    }
+}
+
+describe('Brite Commands', () => {
+    let runner: BriteCommandRunner;
+    const env = process.env.NODE_ENV;
+
+    beforeEach(() => {
+        // Temporarily delete just to make sure we can actually set it
```

Comment:
Can we add a test that checks default/overriding as well.

Reply:
That's what this test does no?